rfc/README.md

```diff
@@ -119,11 +119,12 @@ The list of all RFCs can be found here.
 | 81         | [Log Compaction with Merge Sort](./rfc-81/rfc-81.md)                                                                                                                                                                 | `UNDER REVIEW` |
 | 82         | [Concurrent schema evolution detection](./rfc-82/rfc-82.md)                                                                                                                                                          | `UNDER REVIEW` |
 | 83         | [Incremental Table Service](./rfc-83/rfc-83.md)                                                                                                                                                                      | `COMPLETED`    |
-| 84         | [Optimized SerDe of `DataStream` in Flink operators](./rfc-84/rfc-84.md)                                                                                                                                             | `COMPLETED` |
+| 84         | [Optimized SerDe of `DataStream` in Flink operators](./rfc-84/rfc-84.md)                                                                                                                                             | `COMPLETED`    |
```

Comment:
I think my IDE optimized this line automatically...but it shouldn't affect the markdown visual